src/ts/Search/Select.tsx

```diff
@@ -1,134 +1,166 @@
 import * as React from 'react';
-import { FontIcon } from 'material-ui';
-import { default as SuperSelect } from 'material-ui-superselectfield';
+import * as _ from 'lodash';
+import {
+  FontIcon,
+  SelectField,
+  MenuItem,
+  Toggle,
+} from 'material-ui';
 import debug from 'debug';
 const log = debug('app:Select');
 import i from '../i18n';
 
 export default class Select extends React.Component<any, any> {
-  private dataSource = [];
-  constructor(props) {
-    super(props);
-    // select keeps values chosen by user
-    this.state = { value: [] };
-    this.onChange = this.onChange.bind(this);
+  state = {
```

Comment:
I don't need `constructor` to perform any other operation, so I define `state` as prop, it's compile to the same code: http://stackoverflow.com/questions/37788342